Main-bigint/Main-bigint/main.cpp

```diff
@@ -0,0 +1,90 @@
+#include <iostream>
+#include <fstream>
+
+#include <math.h>
+#include <string>
+#include <vector>
+#include <bitset>
+
+#include "bigint.h"
+#include "calculate.h"
+
+void rwConsole()
+{
+    // Read and write value with console
+    BigInteger bigInt2 = readConsole();
+    writeConsole(bigInt2);
+}
+
+void rwTextFile(const std::string &inputFile, const std::string &outputFile)
+{
+    std::ifstream textFile(inputFile, std::ios::in);
+    std::string line;
+    if (textFile.is_open())
+        getline(textFile, line);
+    else
+    {
+        std::cout << "rwTextFile(): Cannot open file/ File does not exist.\n";
+        return;
+    }
+
+    int numbersOfBigInt = stoi(line);
+    textFile.close();
+
+    // Read and write value with text file
+    BigInteger* listOfBigInt = new BigInteger[numbersOfBigInt];
+    readTextFile(inputFile, listOfBigInt, numbersOfBigInt);
```

Comment:
I think we can restructure the function to only open the inputFile once.

Reply:
It throws an error if I don't initialized `listOfBigInt` before calling the `readTextFile` function.